gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -64,10 +64,10 @@ def get_shared_images
   end
 
   def get_public_images
+    filters = @options.public_images_filters
```

Comment:
@durandom https://gist.github.com/romaintb/f9116cd98d34c743551645d9e5d33433 I am getting stuck with this, if you have further ideas on how to test this, I am listening :)